packages/open-api-page/index.jsx

```diff
@@ -44,7 +43,7 @@ function OpenApiPage({
       <DocsSidenav
         product={productSlug}
         Link={Link}
-        currentPath={isSingleService ? navData[0].path : currentPath}
+        currentPath={currentPath}
```

Comment:
The `isSingleService` prop is no longer needed, `currentPath` will now match as expected due to changes in `server.js` which fix the underlying URL issue.